app/services/container_dashboard_service.rb

```diff
@@ -20,8 +20,8 @@ def all_data
   end
 
   def status
-    if @ems.present? && @ems.kind_of?(ManageIQ::Providers::Openshift::ContainerManager)
-      routes_count = @ems.container_routes.count
+    if @ems.present?
+      routes_count = @ems.respond_to?(:container_routes) ? @ems.container_routes.count : 0 # ems might not have routes
```

Comment:
@zeari why not unify the 'if' to have 2 conditions with: @ems.present? && @ems.respond_to?
also is it necessary to set the zero the way it is set? won't it just return zero count from db on its own when there are no routes?